node/src/env0-deploy-cli.js

```diff
@@ -18,32 +19,25 @@ const {
   ENVIRONMENT_NAME,
   ENVIRONMENT_VARIABLES,
   SENSITIVE_ENVIRONMENT_VARIABLES,
-  REVISION,
-  ARCHIVE_AFTER_DESTROY
 } = OPTIONS;
 
-const optionDefinitions = [
-  { name: API_KEY, alias: 'k', type: String, description: 'Your env0 API Key' },
-  { name: API_SECRET, alias: 's', type: String, description: 'Your env0 API Secret' },
-  { name: ORGANIZATION_ID, alias: 'o', type: String, description: 'Your env0 Organization id' },
-  { name: PROJECT_ID, alias: 'p', type: String, description: 'Your env0 Project id' },
-  { name: BLUEPRINT_ID, alias: 'b', type: String, description: 'The Blueprint id you would like to deploy' },
-  { name: ENVIRONMENT_NAME, alias: 'e', type: String, description: 'The environment name you would like to create, if it exists it will deploy to that environment' },
-  { name: ENVIRONMENT_VARIABLES, alias: 'v', type: String, multiple: true, defaultValue: [], description: 'The environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"' },
-  { name: SENSITIVE_ENVIRONMENT_VARIABLES, alias: 'q', type: String, multiple: true, defaultValue: [], description: 'The sensitive environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"' },
-  { name: REVISION, alias: 'r', type: String, defaultValue: 'master', description: 'Your git revision, can be a branch tag or a commit hash. Default value "master" ' },
-  { name: ARCHIVE_AFTER_DESTROY, type: Boolean, defaultValue: false, description: 'Archive the environment after a successful destroy' },
-];
-
 const availableCommands = {
   'deploy': {
-    options: optionDefinitions,
+    options: allArguments,
```

Comment:
Does this work? This used to be an object and now it's an array

Reply:
It was an array even before 🤭